google-cloud-tasks/samples/test/task_test.rb

```diff
@@ -0,0 +1,50 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "minitest/autorun"
+require "rack/test"
+require "google/cloud/tasks/v2"
+require_relative "../create_http_task.rb"
+require "cgi"
+
+describe "CloudTasks" do
+  include Rack::Test::Methods
+
+  before do
+    @project    = ENV["GOOGLE_CLOUD_PROJECT"]
+    location_id = ENV["LOCATION_ID"] || "us-east1"
```

Comment:
Can you have this default to `us-central1` instead? The GAE app for the project we use as a CI fixture is located in `us-central1`.